content/translations.js

```diff
@@ -0,0 +1,45 @@
+const { findAll } = require("./document");
+
+const TRANSLATIONS_OF = new Map();
+
+function gatherTranslations() {
+  const iter = findAll().iter();
+  for (const {
+    metadata: { slug, locale, title },
+    url,
+  } of iter) {
+    const translation = {
+      title,
+      url,
```

Comment:
We can delete this soon. Right? Because the language drop-down in the client will be able to just patch the URL something like this:
```
for (translation of translations) {
  const translationURL = replaceLocale(currentLocale, translation.locale);
} 
```

Reply:
yes